auditbeat/module/file_integrity/eventreader_test.go

```diff
@@ -405,6 +405,7 @@ func changeGID(t testing.TB, file string) int {
 
 // mustRun runs a sub-test and stops the execution of the parent if the sub-test
 // fails.
+//lint:ignore U1000 Used in unconditionally skipped flakey tests.
```

Comment:
IIUC this is a comment for `staticcheck`. If we decide to standardize on golangci-lint then will this need to change to the `//nolint:staticcheck // <comment>` format?

Reply:
Yes, this needs discussion. It would be good if golangci-lint honoured its dependencies' linter directives (this is golangci/golangci-lint#1658). In the past the coverage of lint classes by staticcheck was quite a lot less than for the metalinters, this is less true now ([vim-go defaults now to staticcheck](https://github.com/fatih/vim-go/pull/3126) now and gopls integrates it ([currently experimentally](https://github.com/golang/tools/blob/master/gopls/doc/settings.md#staticcheck-bool)), so when incremental linting is possible with staticcheck I think that is would be the better option (also for the reason of the default-on .so loading that golangci-lint does). These were in my thinking when I wrote this, with the view that it could be changed down the track if an alternative decision were made.